board/safety/safety_subaru.h

```diff
@@ -20,6 +20,14 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .min_gas = 0,
+  .max_gas = 3400,
+  .min_rpm = 0,
+  .max_rpm = 3200,
+  .max_brake = 400,
```

Comment:
Same, is this -4 m/s^2?

Reply:
actual brake pressure. I will do some plots to see how brake and throttle relates to acceleration

Reply:
brake is pretty linear. 400 is about -2.5m/s^2 according to c3 accelerometer 
![image](https://github.com/commaai/panda/assets/9648890/be716a60-e373-4fe9-b51e-3ab781e406c9)